### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by katiasuya**

### CODE REVIEW

File: internal/repository/model/models.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package model
```
Comment (papahmsolo) [inline internal/repository/model/models.go:1]: Not sure if we it makes sense to separate model, but even if it does, it probably don't deserve a dedicated package

File: internal/repository/model/models.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+import "time"
+
+type User struct {
```
Comment (papahmsolo) [inline internal/repository/model/models.go:5]: Comments?

File: internal/storage/storage.go [hunk 3]
```diff
@@ -55,18 +55,14 @@ func NewS3Client(conf *config.AWSData) (*Storage, error) {
 
 // UploadFile uploads request file.
 func (s *Storage) UploadFile(sourceFile io.Reader, format string) (string, error) {
-	fileID, err := uuid.NewRandom()
-	if err != nil {
-		return "", fmt.Errorf("can't generate file uuid, %w", err)
-	}
-	fileIDStr := fileID.String()
+	fileID := uuid.NewString()
```
Comment (papahmsolo) [inline internal/storage/storage.go:58]: We probably can leave it like this, but this method could panic

File: internal/repository/queries_test.go [hunk 4]
```diff
@@ -0,0 +1,447 @@
...
+				Status:       "queued",
+			},
+			prepare: func(r model.Request) {
+				mock.ExpectQuery(`SELECT r.id, a.name, r.source_format, r.target_format, r.created, r.updated, r.status
+				FROM converter.request r JOIN converter.audio a ON a.id = r.source_id WHERE r.user_id=?`).
+					WithArgs(r.UserID).
+					WillReturnRows(sqlmock.NewRows([]string{"id", "name", "source_format", "target_format", "created", "updated", "status"}).
+						AddRow(r.ID, r.AudioName, r.SourceFormat, r.TargetFormat, r.Created, r.Updated, r.Status))
+			},
+		},
```
Comment (papahmsolo) [inline internal/repository/queries_test.go:136]: Let's add another case or modify the existing test to handle more than one conversion request, you can also test some NULL values returned from DB

File: internal/repository/queries_test.go [hunk 5]
```diff
@@ -0,0 +1,447 @@
...
+			gotRequest, err := repo.GetRequestHistory(tc.expectedHistory.UserID)
+			if !errors.Is(err, tc.expectedError) {
+				t.Errorf("expected error to be %v, but got %v", tc.expectedError, err)
+			}
+
+			if len(gotRequest) > 0 {
+				if tc.expectedHistory != gotRequest[0] {
+					t.Errorf("expected user to be %+v, but got %+v", tc.expectedHistory, gotRequest)
+				}
+			}
```
Comment (papahmsolo) [inline internal/repository/queries_test.go:161]: We can match all expected conversion requests against the returned ones

**Review by papahmsolo — CHANGES_REQUESTED** [review]
Tests look good!

It's a matter of taste but `queries.go` does not contain only queries, so it can live inside `repository.go`

File: internal/repository/model/models.go (hunk 1 above)
Comment (katiasuya) [reply internal/repository/model/models.go:1]: You're right, with the current architecture of the project it's not necessary anymore.

File: internal/storage/storage.go (hunk 3 above)
Comment (katiasuya) [reply internal/storage/storage.go:58]: My bad, I didn't notice that in the description.

File: internal/repository/queries_test.go (hunk 4 above)
Comment (katiasuya) [reply internal/repository/queries_test.go]: The only value that doesn't have "NOT NULL" constraint is targetID and it's not included in request history.

**Review by papahmsolo — APPROVED** [review]
